SampleApps/TestApp/ContentView.swift

```diff
@@ -245,23 +255,35 @@ struct CustomIdentifierView: View {
         VStack {
             VStack {
                 TextField("Enter Identifier", text: $identityItemText)
+                    #if os(iOS)
                     .textFieldStyle(RoundedBorderTextFieldStyle())
+                    #endif
                     .fixedSize()
                     .autocapitalization(.none)
 
                 HStack {
                     TextField("namespace", text: $identityNamespaceText)
+                        #if os(iOS)
```

Comment:
Yes, even I don't like this but then we will loose the styling for iOS as well. Because we would complicate the logic for just for cosmetic enhancement